v0/destinations/facebook_pixel/transform.js

```diff
@@ -29,9 +29,12 @@ const {
  */
 
 const formatRevenue = revenue => {
-  return Number((revenue || 0).toFixed(2));
+  const formattedRevenue = Number(Number(revenue || 0).toFixed(2));
```

Comment:
```suggestion
  const formattedRevenue = Number(revenue || 0).toFixed(2);
```
Why not this?

Reply:
We cannot do this because `toFixed` function returns only a string, not a number. And we need a number @utsabc

Reply:
<img width="242" alt="Screenshot 2022-03-04 at 10 23 21 AM" src="https://user-images.githubusercontent.com/25389167/156701711-93a3cebf-ea9c-4a50-a0c6-fcaf0f6f6675.png">

If we are wrapping `Number(input).toFixed(2)` to `Number(input)` it defeats the purpose of using `toFixed` as the final result doesnot have the 2 decimal place precision

Also Number() is not safe to use as it can give different outcomes at different browser lets use parseFloat

Reply:
done